src/MAUI/Maui.Samples/Samples/Data/EditFeaturesUsingFeatureForms/EditFeaturesUsingFeatureForms.xaml.cs

```diff
@@ -0,0 +1,131 @@
+            }
+        }
+
+        private async void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Maui.GeoViewInputEventArgs e)
+        {
+            try
+            {
+                // Perform identify operation to get the feature
+                var identifyResult = await MyMapView.IdentifyLayersAsync(e.Position, 12, false);
+                var feature = identifyResult.SelectMany(result => result.GeoElements).OfType<ArcGISFeature>().FirstOrDefault();
+
+                if (feature != null)
+                {
+                    // Create a feature form
+                    _featureForm = new FeatureForm(feature);
+
+                    // Add the feature form view to the scroll viewer
+                    FeatureFormViewPanel.FeatureForm = _featureForm;
+
+                    // Show the feature form panel
+                    FeatureFormPanel.IsVisible = true;
+                }
+            }
+            catch (Exception ex)
+            {
+                await DisplayAlert("Error", ex.Message, "OK");
+            }
+        }
+
+        private async void SaveButton_Clicked(object sender, EventArgs e)
+        {
+            try
+            {
+                // Check if there are validation errors
+                if (_featureForm.ValidationErrors.Any())
+                {
+                    var errors = _featureForm.ValidationErrors;
+                    var errorMessages = errors.SelectMany(kvp => kvp.Value.Select(ex => $"{kvp.Key}: {ex.Message}"));
+                    string errorMessage = string.Join("\n", errorMessages);
+                    throw new Exception($"Validation errors exist.\n{errorMessage}");
+                }
+
+                // Finish editing
+                await _featureForm.FinishEditingAsync();
+
+                // Get the service feature table
+                var serviceFeatureTable = (ServiceFeatureTable)_featureForm.Feature.FeatureTable;
+
+                // Get the service geodatabase
+                var serviceGeodatabase = serviceFeatureTable.ServiceGeodatabase;
+
+                // Check if the service geodatabase can apply edits
+                if (serviceGeodatabase.ServiceInfo.CanUseServiceGeodatabaseApplyEdits)
+                {
+                    // Apply edits to the service geodatabase
+                    await serviceGeodatabase.ApplyEditsAsync();
+                }
+                else
+                {
+                    // Apply edits to the service feature table
+                    await serviceFeatureTable.ApplyEditsAsync();
+                }
+                FeatureFormPanel.IsVisible = false;
+            }
+            catch (Exception ex)
+            {
+                await DisplayAlert("Error", ex.Message, "OK");
+            }
+        }
+
+        private void CancelButton_Clicked(object sender, EventArgs e)
+        {
+            // Cancel editing
+            _featureForm.DiscardEdits();
+
+            // Hide the feature form panel
```

Comment:
We try to include full stops at the end of comments in the new samples we create and when we update older samples we add the full stops in.